packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md

```diff
@@ -286,6 +289,7 @@ This endpoint returns an object with the following fields and a `200` code on a
 | `shared_standard_user_name`   | string | (Optional) The name for a shared standard user on Servers in this Project. If `force_shared_ssh_users` is `true`, this must be provided. |
 | `ssh_session_recording`   | boolean | (Optional) Whether to enable ssh recording on all Servers in this Project. Default is `false`. |
 | `team`   | string | The ASA Team of the Project |
+| `user_on_demand_period`   | number | (Optional) The time period, in seconds, that an on-demand user account exists on the server following an access request. By default, this  value is `null` and the on-demand user feature is disabled. |
```

Comment:
Nit: appears there's an extra space in "this  value"

Reply:
Otherwise looks good to me.

Reply:
Good eye! I'll fix that space.